### ISSUE TITLE
Release Files can't be used

### ISSUE THREAD

**Issue by hellc3c** [rll-research/sim2seg#1]
1. the link of "Sim2Seg segmentation model" is the same as the link of the environment resources file, and I can't find any nn model-like file in that folder.
2. the environment resources files have different names from your code in buggy.yaml. I have never used ml-agents library to train RL agents, so I can't configure that yaml file straight. please give a more detailed variables replacement annotation, that would be thankful.

**Comment by hellc3c** [comment]
I find out how to run the sim and finally run the train.py successfully with the `use_s2s=False` option. But the first problem remains.
I assume you are willing to release the pre-trained sim2seg model, In that case, please let me know once you put the right link on the readme.md.

**Comment by typoverflow** [comment]
Also having trouble running the code here. The pre-trained segmentation model is nowhere to be found, so I just set `use_s2s=False` for debugging. I can't figure out how to launch the BuggyGCEnv either -- it says missing executable files (see the following error logs). However, the released files in google drive cannot match the required file path. Any instructions on this? 
Thanks for you time. 

<img width="881" alt="image" src="https://user-images.githubusercontent.com/41679605/203730638-e76c1303-2b8d-43a5-adf3-53cc3b3ae071.png">

**Comment by amberxie88** [comment]
@typoverflow please change `self.BASE_FOLDER` here! https://github.com/rll-research/sim2seg/blob/master/train.py#L72

I'll try and upload a s2s model to the drive link in the coming weeks, sorry for the delay! The segmentation model will be one trained on our Unity environments, so no guarantee it will work perfectly for this simple environment. For now, you should be able to run with `use_s2s=False` per @hellc3c 

If you would like to train a segmentation model on our provided Unity environment, you can collect paired randomized data and segmentation masks through a separate script. You would need to modify the Unity environment itself (C++ code, need to install Unity, etc.) and run a data collection-specific script. Once you have generated the dataset, you can train a segmentation model by making modifications to https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix

Feel free to let me know if there are any other concerns with this codebase or ml-agents!

**Comment by amberxie88** [comment]
Uploaded the model and a corresponding tutorial script: https://github.com/rll-research/sim2seg/blob/master/sim2seg_segmentation.ipynb

Please raise new issues if you come across anything else. Thanks!
